CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+v1.9.4 [unreleased]
+-	[#21890](https://github.com/influxdata/influxdb/pull/21890): chore: update protobuf library versions and remove influx_tsm
```

Comment:
This number should get updated, right?

Reply:
It is updated - this is https://github.com/influxdata/influxdb/pull/21890

Reply:
Ugh, I saw the number in the PR title instead of the grayed-out one next to it...